source/configuration_manual/virtual_attachments_plugin.rst

```diff
@@ -0,0 +1,78 @@
+.. _virtual_attachments_plugin:
+
+==========================
+Virtual Attachments Plugin
+==========================
+
+The virtual-attachments plugin is part of OX Dovecot Pro (Pro only).
+
+Virtual attachments
+===================
+
+First, you'll have to load the plugin which also requires the :ref:`virtual_plugin`
+to be loaded:
+
+.. code-block:: none
+
+  mail_plugins = $mail_plugins virtual virtual_attachments
+
+Namespace configuration
+=======================
+
+Then, you'll have to create a :ref:`namespaces` for the virtual-attachments,
+for example:
+
+.. code-block:: none
+
+  namespace virtual {
+    prefix = virtual/
+    separator = /
+    location = virtual:/etc/dovecot/virtual:INDEX=~/virtual
+
+    mailbox All {
+        auto = create
+    }
```

Comment:
This isn't needed / won't do anything useful. The All mailbox needs to be explicitly configured in filesystem.